spec/models/assignment_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+describe Assignment do
+  describe "validations" do
+    it { should validate_presence_of(:name) }
+  end
+
+  describe "associations" do
+    it { should belong_to(:outcome_coverage) }
+    it { should have_one(:outcome).through(:outcome_coverage) }
+  end
+end
```

Comment:
These tests might be not be needed. I noticed that `assessment_spec.rb` tested the validations, so I added these tests to be consistent. Happy to remove them.